README.md

```diff
@@ -148,8 +144,8 @@ This code sample shows the general usage...
 
 1. If you are going to use rate limiting...
```

Comment:
It's zero or one rate limiter per batcher, right? This makes it sound like a batcher could/would use multiple limiters.

Reply:
Adjusted wording to be clear. Thanks @KeithJRome